app/views/pages/_irc_staff.html.slim

```diff
@@ -14,23 +20,28 @@ ul
 h2#operators Operators
 
 ul
-  li adaedra (Adaedra)
-  li Aria (Aria)
-  li banisterfiend (banisterfiend)
-  li baweaver (baweaver)
-  li havenwood (Havenn)
-  li imperator (rubyhacker1)
-  li jhass (jhass)
-  li ljarvis (ljarvis)
-  li miah (miah)
-  li Mon_Ouie (Mon_Ouie)
-  li seanstickle (seanstickle)
-  li slyphon (slyphon)
-  li smathy (smathy)
-  li workmad3 (workmad3)
-  li zenspider (zenspider)
+  li banisterfiend (banisterfiend) (Freenode)
```

Comment:
Add me?

Reply:
I wanted to do it but lost track of while checking the status of all the others. Sorry :see_no_evil: :sweat_smile: